helm/cmsweb/dbs-migrate/values.yaml

```diff
@@ -0,0 +1,154 @@
+# Default values for dbs-migrate.yaml.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+shareProcessNamespace: true
+
+replicaCount: 
+  prod: 5
+  test: 1
+
+image:
+  repository: cmssw/dbs
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "HG2108h-v2"
+  command:
+  - /bin/bash
+  - /opt/setup-certs-and-run/setup-certs-and-run.sh
+
+environment:
+  prod: false
+  preprod: false
+  test : false
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: 
+  prometheus.io/scrape: 'true'
+  prometheus.io/port: "18257"
+# prometheus.io/port: "19257"
+
+podSecurityContext:
+  runAsGroup: 0
+  runAsUser: 0
+
+securityContext:
+  privileged: true
+
+livenessProbe:
+  exec:
+    command:
+    - cmsweb-ping
+    - "--url=http://localhost:8257/dbs/prod/global/DBSMigrate/"
+    - "--authz=/etc/hmac/hmac"
+    - -verbose
+    - "0"
+  initialDelaySeconds: 120
+  periodSeconds: 10
+
+livenessProbePreProd:
+  exec:
+    command:
+    - cmsweb-ping
+    - "--url=http://localhost:8257/dbs/init/global/DBSMigrate/"
```

Comment:
The URL is not correct. It should be `http://localhost:8257/dbs/int/global/DBSMigrate/`